components/dropdown/dropdown-popover-mixin.js

```diff
@@ -0,0 +1,342 @@
+import { css, html } from 'lit';
+import { classMap } from 'lit/directives/class-map.js';
+import { LocalizeCoreElement } from '../../helpers/localize-core-element.js';
+import { PopoverMixin } from '../popover/popover-mixin.js';
+import { styleMap } from 'lit/directives/style-map.js';
+
+export const DropdownPopoverMixin = superclass => class extends LocalizeCoreElement(PopoverMixin(superclass)) {
+
+	static get properties() {
```

Comment:
This is the identical API as provided by `DropdownContentMixin`, with the exception of boundary as mentioned in the PR description.